src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
+        break;
+      case 'er':
+        ending = info.singular ? ['o', 'es', 'e'] : ['emos', 'eis', 'em'];
+        break;
+      case 'ir':
+        ending = info.singular ? ['o', 'es', 'e'] : ending = ['imos', 'is', 'em'];
+        break;
+      default:
+        break;
+    }
+    return word.slice(0, -2) + ending[parseInt(info.person, 10) - 1];
```

Comment:
please use string builders: ``${word.slice(0, -2)}${ending....}``
instead of quotes, use backticks: https://developers.google.com/web/updates/2015/01/ES6-Template-Strings